assets/js/components/NewBadge.stories.js

```diff
@@ -60,7 +60,7 @@ export default {
 					display: 'flex',
 					justifyContent: 'center',
 					alignItems: 'flex-end',
-					height: '200px',
+					height: '250px',
```

Comment:
Is this necessary? If so that's fine, but it does make it a bit harder to diff the changes.

Reply:
Reverted after making other changes.